### PR TITLE
Update outdated files in dev-1.18-ko.7

### LINKED ISSUE #22097: Outdated files in dev-1.18-ko.7
# This is a Bug Report
## Problem
Outdated files in the upstream/dev-1.18-ko.7 branch.

### 59 files to be modified 
1. [ ] content/en/community/_index.html 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/_index.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/cluster-administration/cloud-providers.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/configuration/configmap.md 1 file changed, 6 insertions(+), 6 deletions(-)
1. [ ] content/en/docs/concepts/configuration/overview.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/configuration/resource-bin-packing.md 1 file changed, 9 insertions(+), 9 deletions(-)
1. [ ] content/en/docs/concepts/containers/images.md 1 file changed, 93 insertions(+), 166 deletions(-)
1. [ ] content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/extend-kubernetes/extend-cluster.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/overview/kubernetes-api.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/overview/what-is-kubernetes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/overview/working-with-objects/namespaces.md 1 file changed, 7 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/overview/working-with-objects/object-management.md 1 file changed, 6 deletions(-)
1. [ ] content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/security/overview.md 1 file changed, 75 insertions(+), 86 deletions(-)
1. [ ] content/en/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md 1 file changed, 30 insertions(+), 29 deletions(-)
1. [ ] content/en/docs/concepts/storage/volume-snapshot-classes.md 1 file changed, 25 insertions(+), 12 deletions(-)
1. [ ] content/en/docs/concepts/storage/volumes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/controllers/cron-jobs.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 12 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/workloads/controllers/garbage-collection.md 1 file changed, 1 insertion(+), 9 deletions(-)
1. [ ] content/en/docs/concepts/workloads/controllers/ttlafterfinished.md 1 file changed, 4 insertions(+), 4 deletions(-)
1. [ ] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 2 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/contribute/advanced.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/contribute/new-content/open-a-pr.md 1 file changed, 19 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/home/_index.md 1 file changed, 2 insertions(+)
1. [ ] content/en/docs/reference/glossary/cronjob.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/reference/glossary/job.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/reference/kubectl/cheatsheet.md 1 file changed, 3 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/reference/kubectl/overview.md 1 file changed, 43 insertions(+), 32 deletions(-)
1. [ ] content/en/docs/setup/best-practices/cluster-large.md 1 file changed, 3 deletions(-)
1. [ ] content/en/docs/setup/best-practices/node-conformance.md 1 file changed, 1 deletion(-)
1. [ ] content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 15 insertions(+), 6 deletions(-)
1. [ ] content/en/docs/tasks/_index.md 1 file changed, 65 deletions(-)
1. [ ] content/en/docs/tasks/access-application-cluster/_index.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/tasks/administer-cluster/_index.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/configure-pod-container/_index.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/tasks/debug-application-cluster/_index.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/tasks/extend-kubectl/kubectl-plugins.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/inject-data-application/_index.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/tasks/manage-daemon/_index.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/tasks/manage-daemon/rollback-daemon-set.md 1 file changed, 33 insertions(+), 44 deletions(-)
1. [ ] content/en/docs/tasks/manage-gpus/scheduling-gpus.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md 1 file changed, 2 insertions(+), 12 deletions(-)
1. [ ] content/en/docs/tasks/manage-kubernetes-objects/_index.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/tasks/network/_index.md 1 file changed, 2 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/run-application/_index.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/tasks/run-application/horizontal-pod-autoscale.md 1 file changed, 2 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/tls/_index.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/tasks/tools/_index.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/tasks/tools/install-kubectl.md 1 file changed, 121 insertions(+), 120 deletions(-)
1. [ ] content/en/docs/tutorials/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tutorials/configuration/configure-redis-using-configmap.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tutorials/stateful-application/basic-stateful-set.md 1 file changed, 407 insertions(+), 254 deletions(-)
1. [ ] content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md 1 file changed, 3 insertions(+), 3 deletions(-)
1. [ ] content/en/partners/_index.html 1 file changed, 67 insertions(+), 56 deletions(-)

### 1 files to be renamed  
1. [ ] R097 content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md -> content/en/docs/concepts/workloads/controllers/job.md

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `upstream/dev-1.18-ko.7` branch.

For example:
```
# checkout `upstream/dev-1.18-ko.7`
...
# check what is updated in the upstream 
git diff upstream/dev-1.18-ko.6 upstream/dev-1.18-ko.7 -- content/en/community/_index.html
# apply changes to content/ko
vi content/ko/community/_index.html
...
# commit and push
...
# make PR to `upstream/dev-1.18-ko.7`
```

## Pages to Update

/language ko

### PR DESCRIPTION
**Pull request by pjhwa**
Closes #22097 

 X content/en/community/_index.html 1 file changed, 1 insertion(+), 1 deletion(-): 이미 반영되어있음
 O content/en/docs/concepts/_index.md 1 file changed, 2 insertions(+), 2 deletions(-)
 O content/en/docs/concepts/cluster-administration/cloud-providers.md 1 file changed, 2 insertions(+), 2 deletions(-)
 O content/en/docs/concepts/configuration/configmap.md 1 file changed, 6 insertions(+), 6 deletions(-)
 O content/en/docs/concepts/configuration/overview.md 1 file changed, 2 insertions(+), 2 deletions(-)
 O content/en/docs/concepts/configuration/resource-bin-packing.md 1 file changed, 9 insertions(+), 9 deletions(-)
 O content/en/docs/concepts/containers/images.md 1 file changed, 93 insertions(+), 166 deletions(-)
 X content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md 1 file changed, 1 insertion(+), 1 deletion(-): 이미 반영되어있음
 X content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md 1 file changed, 1 insertion(+), 1 deletion(-): 이미 반영되어있음
 O content/en/docs/concepts/extend-kubernetes/extend-cluster.md 1 file changed, 1 insertion(+), 1 deletion(-)
 X content/en/docs/concepts/overview/kubernetes-api.md 1 file changed, 1 insertion(+), 1 deletion(-): 이미 반영되어있음
 O content/en/docs/concepts/overview/what-is-kubernetes.md 1 file changed, 1 insertion(+), 1 deletion(-)
 O content/en/docs/concepts/overview/working-with-objects/namespaces.md 1 file changed, 7 insertions(+), 2 deletions(-)
 O content/en/docs/concepts/overview/working-with-objects/object-management.md 1 file changed, 6 deletions(-)
 O content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md 1 file changed, 1 insertion(+), 1 deletion(-)
 O content/en/docs/concepts/security/overview.md 1 file changed, 75 insertions(+), 86 deletions(-)
 O content/en/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md 1 file changed, 30 insertions(+), 29 deletions(-)
 O content/en/docs/concepts/storage/volume-snapshot-classes.md 1 file changed, 25 insertions(+), 12 deletions(-)
 O content/en/docs/concepts/storage/volumes.md 1 file changed, 1 insertion(+), 1 deletion(-)
 X content/en/docs/concepts/workloads/controllers/cron-jobs.md 1 file changed, 1 insertion(+), 1 deletion(-): 이미 반영되어있음
 O content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 12 insertions(+), 2 deletions(-)
 O content/en/docs/concepts/workloads/controllers/garbage-collection.md 1 file changed, 1 insertion(+), 9 deletions(-)
 O content/en/docs/concepts/workloads/controllers/ttlafterfinished.md 1 file changed, 4 insertions(+), 4 deletions(-)
 O content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 2 insertions(+), 3 deletions(-)
 O content/en/docs/contribute/advanced.md 1 file changed, 1 insertion(+)
 O content/en/docs/contribute/new-content/open-a-pr.md 1 file changed, 19 insertions(+), 3 deletions(-)
 O content/en/docs/home/_index.md 1 file changed, 2 insertions(+)
 O content/en/docs/reference/glossary/cronjob.md 1 file changed, 1 insertion(+), 1 deletion(-)
 O content/en/docs/reference/glossary/job.md 1 file changed, 1 insertion(+), 1 deletion(-)
 O content/en/docs/reference/kubectl/cheatsheet.md 1 file changed, 3 insertions(+), 3 deletions(-)
 O content/en/docs/reference/kubectl/overview.md 1 file changed, 43 insertions(+), 32 deletions(-)
 O content/en/docs/setup/best-practices/cluster-large.md 1 file changed, 3 deletions(-)
 O content/en/docs/setup/best-practices/node-conformance.md 1 file changed, 1 deletion(-)
 O content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 15 insertions(+), 6 deletions(-)
 O content/en/docs/tasks/_index.md 1 file changed, 65 deletions(-)
 O content/en/docs/tasks/access-application-cluster/_index.md 1 file changed, 1 insertion(+)
 O content/en/docs/tasks/administer-cluster/_index.md 1 file changed, 1 insertion(+)
 O content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md 1 file changed, 1 insertion(+), 1 deletion(-)
 O content/en/docs/tasks/configure-pod-container/_index.md 1 file changed, 1 insertion(+)
 O content/en/docs/tasks/debug-application-cluster/_index.md 1 file changed, 1 insertion(+)
 O content/en/docs/tasks/extend-kubectl/kubectl-plugins.md 1 file changed, 1 insertion(+), 1 deletion(-)
 O content/en/docs/tasks/inject-data-application/_index.md 1 file changed, 1 insertion(+)
 O content/en/docs/tasks/manage-daemon/_index.md 1 file changed, 1 insertion(+)
 O content/en/docs/tasks/manage-daemon/rollback-daemon-set.md 1 file changed, 33 insertions(+), 44 deletions(-)
 O content/en/docs/tasks/manage-gpus/scheduling-gpus.md 1 file changed, 1 insertion(+)
 O content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md 1 file changed, 2 insertions(+), 12 deletions(-)
 O content/en/docs/tasks/manage-kubernetes-objects/_index.md 1 file changed, 1 insertion(+)
 O content/en/docs/tasks/network/_index.md 1 file changed, 2 insertions(+), 1 deletion(-)
 O content/en/docs/tasks/run-application/_index.md 1 file changed, 1 insertion(+)
 O content/en/docs/tasks/run-application/horizontal-pod-autoscale.md 1 file changed, 2 insertions(+), 1 deletion(-)
 O content/en/docs/tasks/tls/_index.md 1 file changed, 1 insertion(+)
 O content/en/docs/tasks/tools/install-kubectl.md 1 file changed, 121 insertions(+), 120 deletions(-)
 O content/en/docs/tasks/tools/_index.md 1 file changed, 1 insertion(+)
 O content/en/docs/tutorials/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
 X content/en/docs/tutorials/configuration/configure-redis-using-configmap.md 1 file changed, 1 insertion(+), 1 deletion(-): 이미 반영되어있음
 O content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html 1 file changed, 1 insertion(+), 1 deletion(-)
 O content/en/docs/tutorials/stateful-application/basic-stateful-set.md 1 file changed, 407 insertions(+), 254 deletions(-)
 O content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md 1 file changed, 3 insertions(+), 3 deletions(-)
 O content/en/partners/_index.html 1 file changed, 67 insertions(+), 56 deletions(-)

1 files to be renamed
 O R097 content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md -> content/en/docs/concepts/workloads/controllers/job.md

### CODE REVIEW

**Comment by pjhwa** [comment]
일부만 수정했습니다. 
생각보다 일이 많네요. 작업한 부분만 먼저 PR 올립니다.

**Comment by gochist** [comment]
@pjhwa 감사합니다!
/assign @yoonian

File: content/ko/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -74,139 +79,57 @@ Docker *18.06 또는 그 이상* 을 사용하길 바란다. 더 낮은 버전
...
-
-요구 사항:
-
-- Kubelet 버전 `v1.2.0` 이상을 사용해야 한다.  (예: `/usr/bin/kubelet --version=true`를 실행).
-- 노드가 지역 A에 있고 레지스트리가 다른 지역 B에 있다면, 버전 `v1.3.0` 이상이 필요하다.
-- 사용자의 지역에서 ECR이 지원되어야 한다.
+이들 옵션은 아래에서 더 자세히 설명한다.
 
-문제 해결:
+### 개인 레지스트리에 인증하도록 노드 구성
```
Comment (yoonian) [inline content/ko/docs/concepts/containers/images.md:89]:
```suggestion
### 프라이빗 레지스트리에 인증하도록 노드 구성
```

용어를 통일해서 써야 할듯요.

File: content/ko/docs/concepts/containers/images.md [hunk 2]
```diff
@@ -281,26 +204,28 @@ GCE 및 자동 노드 교체를 수행하는 다른 클라우드 제공자에 
...
-만약 Docker 자격 증명 파일이 이미 존재한다면, 위의 명령을 사용하지 않고,
-자격 증명 파일을 쿠버네티스 시크릿으로 가져올 수 있다.
-[기존 Docker 자격 증명으로 시크릿 생성](/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials)에서 관련 방법을 설명하고 있다.
+만약 도커 자격 증명 파일이 이미 존재한다면, 위의 명령을 사용하지 않고,
+자격 증명 파일을 쿠버네티스 {{< glossary_tooltip text="시크릿" term_id="secret" >}}으로
+가져올 수 있다.
+[기존 도커 자격 증명으로 시크릿 생성](/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials)에서 관련 방법을 설명하고 있다.
+
 `kubectl create secret docker-registry`는
 하나의 개인 레지스트리에서만 작동하는 시크릿을 생성하기 때문에,
```
Comment (yoonian) [inline content/ko/docs/concepts/containers/images.md:227]:
```suggestion
하나의 프라이빗 레지스트리에서만 작동하는 시크릿을 생성하기 때문에,
```

File: content/ko/docs/concepts/overview/what-is-kubernetes.md [hunk 3]
```diff
@@ -71,7 +71,7 @@ card:
 
 ## 쿠버네티스가 아닌 것
 
-쿠버네티스는 전통적인, 모든 것이 포함된 Platform as a Service(PaaS)가 아니다. 쿠버네티스는 하드웨어 수준보다는 컨테이너 수준에서 운영되기 때문에, PaaS가 일반적으로 제공하는 배포, 스케일링, 로드 밸런싱, 로깅 및 모니터링과 같은 기능에서 공통점이 있기도 하다. 하지만, 쿠버네티스는 모놀리식(monolithic)이 아니어서, 이런 기본 솔루션이 선택적이며 추가나 제거가 용이하다. 쿠버네티스는 개발자 플랫폼을 만드는 구성 요소를 제공하지만, 필요한 경우 사용자의 선택권과 유연성을 지켜준다.
+쿠버네티스는 전통적인, 모든 것이 포함된 Platform as a Service(PaaS)가 아니다. 쿠버네티스는 하드웨어 수준보다는 컨테이너 수준에서 운영되기 때문에, PaaS가 일반적으로 제공하는 배포, 확장, 로드 밸런싱과 같은 기능을 제공하며, 사용자가 로깅, 모니터링 및 알림 솔루션을 통합할 수 있다. 하지만, 쿠버네티스는 모놀리식(monolithic)이 아니어서, 이런 기본 솔루션이 선택적이며 추가나 제거가 용이하다. 쿠버네티스는 개발자 플랫폼을 만드는 구성 요소를 제공하지만, 필요한 경우 사용자의 선택권과 유연성을 지켜준다.
```
Comment (yoonian) [inline content/ko/docs/concepts/overview/what-is-kubernetes.md:74]:
```suggestion
쿠버네티스는 전통적인, 모든 것이 포함된 Platform as a Service(PaaS)가 아니다. 쿠버네티스는 하드웨어 수준보다는 컨테이너 수준에서 운영되기 때문에, PaaS가 일반적으로 제공하는 배포, 스케일링, 로드 밸런싱과 같은 기능을 제공하며, 사용자가 로깅, 모니터링 및 알림 솔루션을 통합할 수 있다. 하지만, 쿠버네티스는 모놀리식(monolithic)이 아니어서, 이런 기본 솔루션이 선택적이며 추가나 제거가 용이하다. 쿠버네티스는 개발자 플랫폼을 만드는 구성 요소를 제공하지만, 필요한 경우 사용자의 선택권과 유연성을 지켜준다.
```
확장이라고 하면 뜻이 너무 확장되어서요. ;;;

File: content/ko/docs/concepts/overview/working-with-objects/namespaces.md [hunk 4]
```diff
@@ -54,11 +58,12 @@ kube-public       Active   1d
 kube-system       Active   1d
 ```
 
-쿠버네티스는 처음에 세 개의 초기 네임스페이스를 갖는다.
+쿠버네티스는 처음에 네 개의 초기 네임스페이스를 갖는다.
 
    * `default` 다른 네임스페이스가 없는 오브젝트를 위한 기본 네임스페이스
    * `kube-system` 쿠버네티스 시스템에서 생성한 오브젝트를 위한 네임스페이스
    * `kube-public` 이 네임스페이스는 자동으로 생성되며 모든 사용자(인증되지 않은 사용자 포함)가 읽기 권한으로 접근할 수 있다. 이 네임스페이스는 주로 전체 클러스터 중에 공개적으로 드러나서 읽을 수 있는 리소스를 위해 예약되어 있다. 이 네임스페이스의 공개적인 성격은 단지 관례이지 요구 사항은 아니다.
+   * `kube-node-lease` 클러스터가 확장될 때 노드 하트비트의 성능을 향상시키는 각 노드와 관련된 리스(lease) 오브젝트에 대한 네임스페이스
```
Comment (yoonian) [inline content/ko/docs/concepts/overview/working-with-objects/namespaces.md:66]:
```suggestion
   * `kube-node-lease` 클러스터가 스케일링될 때 노드 하트비트의 성능을 향상시키는 각 노드와 관련된 리스(lease) 오브젝트에 대한 네임스페이스
```
`확장`은 의미가 광범위하게 여겨지고요. 클러스터 내에 노드의 수를 늘리는 `스케일링`이 맞지 않나 싶습니다.

File: content/ko/docs/concepts/security/overview.md [hunk 5]
```diff
@@ -112,51 +109,45 @@ RBAC 인증(쿠버네티스 API에 대한 접근) | https://kubernetes.io/docs/r
...
-* 컨트롤 플레인에 대한 [전송 데이터 암호화](/docs/tasks/tls/managing-tls-in-a-cluster/) 알아보기
-* [Rest에서 데이터 암호화](/docs/tasks/administer-cluster/encrypt-data/) 알아보기
-* [쿠버네티스 시크릿](/docs/concepts/configuration/secret/)에 대해 알아보기
+쿠버네티스 보안 주제에 관련한 내용들을 배워보자.
+
+* [파드 보안 표준](/docs/concepts/security/pod-security-standards/)
+* [파드에 대한 네트워크 정책](/ko/docs/concepts/services-networking/network-policies/)
+* [클러스터 보안](/docs/tasks/administer-cluster/securing-a-cluster/)
+* [API 접근 통제](/docs/reference/access-authn-authz/controlling-access/)
+* [전송 데이터 암호화](/docs/tasks/tls/managing-tls-in-a-cluster/) for the control plane
```
Comment (yoonian) [inline content/ko/docs/concepts/security/overview.md:151]:
```suggestion
* 컨트롤 플레인을 위한 [전송 데이터 암호화](/docs/tasks/tls/managing-tls-in-a-cluster/)
```

빠져 있어서 추가했습니다.

File: content/ko/docs/concepts/storage/volume-snapshot-classes.md [hunk 6]
```diff
@@ -6,7 +6,7 @@ weight: 30
 
 <!-- overview -->
 
-이 문서는 쿠버네티스의 `VolumeSnapshotClass` 개요를 설명한다.
+이 문서는 쿠버네티스의 VolumeSnapshotClass 개요를 설명한다.
```
Comment (yoonian) [inline content/ko/docs/concepts/storage/volume-snapshot-classes.md:9]:
```suggestion
이 문서는 쿠버네티스의 볼륨스냅샷클래스(VolumeSnapshotClass) 개요를 설명한다.
```
요 부분은 고민이 되는데, 제 의견을 일단 드려봅니다.
전체적으로 볼륨스냅샷클래스로 쓰되 첫 표기이니 한영 병기하는 것이 좋을 것 같습니다.

File: content/ko/docs/concepts/storage/volume-snapshot-classes.md [hunk 7]
```diff
@@ -17,29 +17,42 @@ weight: 30
 
 ## 소개
 
-`StorageClass` 는 관리자가 볼륨을 프로비저닝할 때 제공하는 스토리지의 "클래스"를
-설명하는 방법을 제공하는 것처럼, `VolumeSnapshotClass` 는 볼륨 스냅샷을
+스토리지클래스(StorageClass)는 관리자가 볼륨을 프로비저닝할 때 제공하는 스토리지의 "클래스"를
+설명하는 방법을 제공하는 것처럼, VolumeSnapshotClass는 볼륨 스냅샷을
```
Comment (yoonian) [inline content/ko/docs/concepts/storage/volume-snapshot-classes.md:21]:
```suggestion
설명하는 방법을 제공하는 것처럼, 볼륨스냅샷클래스는 볼륨 스냅샷을
```

File: content/ko/docs/concepts/storage/volume-snapshot-classes.md [hunk 8]
```diff
@@ -17,29 +17,42 @@ weight: 30
...
 프로비저닝할 때 스토리지의 "클래스"를 설명하는 방법을 제공한다.
 
 ## VolumeSnapshotClass 리소스
 
-각 `VolumeSnapshotClass` 에는 클래스에 속하는 `VolumeSnapshot` 을
+각 VolumeSnapshotClass에는 클래스에 속하는 VolumeSnapshot을
```
Comment (yoonian) [inline content/ko/docs/concepts/storage/volume-snapshot-classes.md:26]:
```suggestion
각 볼륨스냅샷클래스에는 클래스에 속하는 볼륨스냅샷을
```

File: content/ko/docs/concepts/storage/volume-snapshot-classes.md [hunk 9]
```diff
@@ -17,29 +17,42 @@ weight: 30
...
 동적으로 프로비전 할 때 사용되는 `driver`, `deletionPolicy` 그리고 `parameters`
 필드를 포함한다.
 
-`VolumeSnapshotClass` 오브젝트의 이름은 중요하며, 사용자가 특정
-클래스를 요청할 수 있는 방법이다. 관리자는 `VolumeSnapshotClass` 오브젝트를
+VolumeSnapshotClass 오브젝트의 이름은 중요하며, 사용자가 특정
```
Comment (yoonian) [inline content/ko/docs/concepts/storage/volume-snapshot-classes.md:30]:
```suggestion
볼륨스냅샷클래스 오브젝트의 이름은 중요하며, 사용자가 특정
```

File: content/ko/docs/concepts/storage/volume-snapshot-classes.md [hunk 10]
```diff
@@ -17,29 +17,42 @@ weight: 30
...
+클래스를 요청할 수 있는 방법이다. 관리자는 VolumeSnapshotClass 오브젝트를
```
Comment (yoonian) [inline content/ko/docs/concepts/storage/volume-snapshot-classes.md:31]:
```suggestion
클래스를 요청할 수 있는 방법이다. 관리자는 볼륨스냅샷클래스 오브젝트를
```

File: content/ko/docs/concepts/storage/volume-snapshot-classes.md [hunk 11]
```diff
@@ -17,29 +17,42 @@ weight: 30
...
+metadata:
+  name: csi-hostpath-snapclass
+driver: hostpath.csi.k8s.io
+deletionPolicy: Delete
+parameters:
+```
+
+관리자는`snapshot.storage.kubernetes.io/is-default-class: "true"` 어노테이션을 추가하여
+바인딩할 특정 클래스를 요청하지 않는 볼륨스냅샷에 대한
+기본 VolumeSnapshotClass를 지정할 수 있다.
```
Comment (yoonian) [inline content/ko/docs/concepts/storage/volume-snapshot-classes.md:47]:
```suggestion
기본 볼륨스냅샷클래스를 지정할 수 있다.
```

File: content/ko/docs/tasks/manage-daemon/rollback-daemon-set.md [hunk 12]
```diff
@@ -124,30 +116,29 @@ daemonset "<daemonset-name>" successfully rolled out
 kubectl get controllerrevision -l <daemonset-selector-key>=<daemonset-selector-value>
 ```
 
-이 명령은 `ControllerRevisions` 의 목록을 반환한다.
+이 명령은 ControllerRevisions의 목록을 반환한다.
```
Comment (yoonian) [inline content/ko/docs/tasks/manage-daemon/rollback-daemon-set.md:119]:
```suggestion
이 명령은 ControllerRevision의 목록을 반환한다.
```

File: content/ko/docs/tasks/tools/install-kubectl.md [hunk 13]
```diff
@@ -111,95 +111,95 @@ kubectl version --client
 
 1. 최신 릴리스를 다운로드한다.
 
-    ```
-    curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl"
-    ```
+   ```bash
```
Comment (yoonian) [inline content/ko/docs/tasks/tools/install-kubectl.md:114]: 요기 커멘트는 아니지만, 원문이 그런데, 왜 `bash` 인지 모르겠네요.

커맨드가 `bash` 에 종속된 내용도 아니고,
예전에 ` ```sh` 로 PR 올리면, `sh` 지우거나 `shell` 로 바꾸기도 했었던 것 같은데 말입니다.

File: content/ko/docs/concepts/containers/images.md (hunk 1 above)
Comment (pjhwa) [reply content/ko/docs/concepts/containers/images.md:89]: 반영하겠습니다.

File: content/ko/docs/concepts/containers/images.md (hunk 2 above)
Comment (pjhwa) [reply content/ko/docs/concepts/containers/images.md:227]: 반영하겠습니다.

File: content/ko/docs/concepts/overview/what-is-kubernetes.md (hunk 3 above)
Comment (pjhwa) [reply content/ko/docs/concepts/overview/what-is-kubernetes.md:74]: 반영하겠습니다.

File: content/ko/docs/concepts/overview/working-with-objects/namespaces.md (hunk 4 above)
Comment (pjhwa) [reply content/ko/docs/concepts/overview/working-with-objects/namespaces.md:66]: 반영하겠습니다.

File: content/ko/docs/concepts/security/overview.md (hunk 5 above)
Comment (pjhwa) [reply content/ko/docs/concepts/security/overview.md:151]: 반영하겠습니다.

File: content/ko/docs/concepts/storage/volume-snapshot-classes.md (hunk 6 above)
Comment (pjhwa) [reply content/ko/docs/concepts/storage/volume-snapshot-classes.md:9]: 반영하겠습니다.

File: content/ko/docs/concepts/storage/volume-snapshot-classes.md (hunk 7 above)
Comment (pjhwa) [reply content/ko/docs/concepts/storage/volume-snapshot-classes.md:21]: 반영하겠습니다.

File: content/ko/docs/concepts/storage/volume-snapshot-classes.md (hunk 8 above)
Comment (pjhwa) [reply content/ko/docs/concepts/storage/volume-snapshot-classes.md:26]: 반영하겠습니다.

File: content/ko/docs/concepts/storage/volume-snapshot-classes.md (hunk 9 above)
Comment (pjhwa) [reply content/ko/docs/concepts/storage/volume-snapshot-classes.md:30]: 반영하겠습니다.

File: content/ko/docs/concepts/storage/volume-snapshot-classes.md (hunk 10 above)
Comment (pjhwa) [reply content/ko/docs/concepts/storage/volume-snapshot-classes.md:31]: 반영하겠습니다.

File: content/ko/docs/concepts/storage/volume-snapshot-classes.md (hunk 11 above)
Comment (pjhwa) [reply content/ko/docs/concepts/storage/volume-snapshot-classes.md:47]: 반영하겠습니다.

File: content/ko/docs/tasks/manage-daemon/rollback-daemon-set.md (hunk 12 above)
Comment (pjhwa) [reply content/ko/docs/tasks/manage-daemon/rollback-daemon-set.md:119]: 반영하겠습니다.

File: content/ko/docs/tasks/tools/install-kubectl.md (hunk 13 above)
Comment (pjhwa) [reply content/ko/docs/tasks/tools/install-kubectl.md:114]: 여러 문서를 작업해보다보면, 원문이 사실 일관성이 깨지는 문서들이 좀 있기는 한 것 같습니다.
(앞의 공백도 4칸이던 것을 3칸으로 줄인데는 뭔가 이유가 있을 것 같은데 말입니다. ^^;)

**Comment by pjhwa** [comment]
@yoonian 리뷰 감사합니다.
주신 의견에 대해 반영하였습니다.

**Comment by yoonian** [comment]
@pjhwa 이런! #22148 랑 컨플릭이 발생했네요. 
`content/ko/docs/concepts/configuration/overview.md` 요기 좀 수정 부탁드려요.

**Comment by pjhwa** [comment]
> @pjhwa 이런! #22148 랑 컨플릭이 발생했네요.
> `content/ko/docs/concepts/configuration/overview.md` 요기 좀 수정 부탁드려요.

@yoonian 여기저기 충돌이 발생했네요. ^^ 암튼 수정해서 업데이트했습니다. 감사합니다!

**Comment by yoonian** [comment]
> > @pjhwa 이런! #22148 랑 컨플릭이 발생했네요.
> > `content/ko/docs/concepts/configuration/overview.md` 요기 좀 수정 부탁드려요.
> 
> @yoonian 여기저기 충돌이 발생했네요. ^^ 암튼 수정해서 업데이트했습니다. 감사합니다!

이번엔 컨플릭이 자주 발생하네요. ;; 시간 정해 놓고 재빨리 머지해야겠습니다. ^^

**Comment by pjhwa** [comment]
> > @yoonian 여기저기 충돌이 발생했네요. ^^ 암튼 수정해서 업데이트했습니다. 감사합니다!
> 
> 이번엔 생각지도 않은 에러가... 시간 정해 놓고 재빨리 머지해야겠습니다. ^^

#22222 이슈로 인해 netlify 빌드가 안되네요. 일단 #22223 PR에 맞춰 수정해놓았습니다. (임시 솔루션이라 나중에 master에 맞춰 다시 반영해야할 것 같습니다. - 아직 솔루션이 안나온 상태라...)

**Comment by gochist** [comment]
@pjhwa @yoonian 감사합니다! `/layouts` 디렉토리의 변경은 sig-docs-ko-owners 가 승인할 수 없습니다. #22223 을 포함하지 않아도 현재 빌드가 정상적으로 되어서 #22223 을 revert 하고 승인합니다.

/lgtm
/approve

**Pull request merged**